src/pages/Contact.jsx

```diff
@@ -9,80 +9,90 @@ import {
 } from '../lib/mail';
 import { BRAND_NAME, SUBSCRIPTION_FORM_URL } from '../config/constants';
 import { Container, Link as MuiLink, Typography } from '@mui/material';
+import { useTranslation, Trans } from 'react-i18next';
 
 function Contact() {
+  const { t } = useTranslation();
+
   return (
     <Container>
       <Helmet>
-        <title>Contact</title>
+        <title>{t('contact.titleTag')}</title>
       </Helmet>
 
       <Typography variant="h4" align="center" gutterBottom>
-        Contact Us
+        {t('contact.contactUs')}
       </Typography>
 
       <Typography variant="h5" gutterBottom>
-        Found a bug?
+        {t('contact.foundBug')}
       </Typography>
       <Typography paragraph>
-        You can file it{' '}
-        <MuiLink
-          href={genMailToLink({
-            subject: 'Bug Report',
-            body: withDeviceInfo(reportIssue),
-          })}>
-          here
-        </MuiLink>
-        .
+        <Trans i18nKey="contact.fileBug">
+          You can file it
+          <MuiLink
+            href={genMailToLink({
+              subject: 'Bug Report',
+              body: withDeviceInfo(reportIssue),
+            })}>
+            here
+          </MuiLink>
+        </Trans>
       </Typography>
 
       <Typography variant="h5" gutterBottom>
-        Feature Requests
+        {t('contact.featureReq')}
       </Typography>
       <Typography paragraph>
-        You can suggest an idea for this project{' '}
-        <MuiLink
-          href={genMailToLink({
-            subject: 'Feature Request',
-            body: withDeviceInfo(featureRequest),
-          })}>
-          here
-        </MuiLink>
-        .
+        <Trans i18nKey="contact.featureSuggest">
+          You can suggest an idea for this project
+          <MuiLink
+            href={genMailToLink({
+              subject: 'Feature Request',
+              body: withDeviceInfo(featureRequest),
+            })}>
+            here
+          </MuiLink>
+        </Trans>
       </Typography>
 
       <Typography variant="h5" gutterBottom>
-        Learn About Us
+        {t('contact.learnAboutUs')}
       </Typography>
       <Typography paragraph>
-        You can learn more about {BRAND_NAME} and the developers{' '}
-        <MuiLink component={Link} to="/about">
-          here
-        </MuiLink>
-        .
+        <Trans i18nKey="contact.learnMore" values={{ brand: BRAND_NAME }}>
+          You can learn more about and the developers
```

Comment:
This is the default string if no translation appears right? It's a little different from the English one as the brand name and ending period are missing.

Reply:
I hard coded the brand name so it won't mess the translation up.